enterprise_access/apps/workflow/serialization.py

```diff
@@ -0,0 +1,38 @@
+"""
+Workflow Steps do work on objects, not on dicts.  The BaseInputOutput
+class below can be subclassed by attrs classes, so that every input/output
+class utilized by workflow steps can be counted on to have a from_dict()
+classmethod to generate an object from a dict, and a to_dict() method
+to turn an instance of an input/output class back into a dict (in other words
+to structure and unstructure, or to deserialize and then serialize).
+"""
+import uuid
+
+import attrs
+import cattrs
+
+CONVERTER = cattrs.Converter()
+
+
+@CONVERTER.register_structure_hook
+def uuid_structure_hook(val: str, _) -> uuid.UUID:
+    return uuid.UUID(val)
+
+
+@CONVERTER.register_unstructure_hook
+def uuid_unstructure_hook(val: uuid.UUID) -> str:
+    return str(val)
```

Comment:
Nit: Since this is at least my first foray into attrs/cattrs and likely others too, maybe add a code comment here stating that we need to teach cattrs how to structure/destructure UUIDs because it happens to not be supported by default: https://catt.rs/en/stable/defaulthooks.html